smartsim/_core/entrypoints/telemetrymonitor.py

```diff
@@ -599,7 +599,7 @@ def main(
     )
 
     cooldown_duration = cooldown_duration or CONFIG.telemetry_cooldown
-    logger.warning(cooldown_duration)
+    logger.debug(f"Cooldown period (seconds): {cooldown_duration}")
```

Comment:
I need to get 612 merged... There is [some logging](https://github.com/CrayLabs/SmartSim/pull/460/files#diff-7d23cb75a6eb04cb271c5344e1d142d1989fc2960385718d3d4c936d02f38c1dR526) of cooldown progress already in there that might meet your need. 

If that didn't do it, I think another spot would be more ideal

Reply:
This was some legacy debug line I think I should really remove.